packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -794,9 +794,13 @@ export class ContainerRuntime
 
 		const registry = new FluidDataStoreRegistry(registryEntries);
 
+		const baseSnapshotTree =
+			context.snapshotWithContents !== undefined
+				? context.snapshotWithContents.snapshotTree
+				: context.baseSnapshot;
 		const tryFetchBlob = async <T>(blobName: string): Promise<T | undefined> => {
-			const blobId = context.baseSnapshot?.blobs[blobName];
-			if (context.baseSnapshot && blobId) {
+			const blobId = baseSnapshotTree?.blobs[blobName];
+			if (baseSnapshotTree && blobId) {
```

Comment:
I'm curious why this change is needed. I'd think that old code would work, because we would populate both structures.
It might be good long-term, but I'd rather wait for new loader saturation, and then completely remove old code paths (i.e. remove context.baseSnapshot), and with that - find and patch all the places like that.

Reply:
Removed.

Reply:
Just wanted to use the new structure.